frontend/taipy-gui/src/components/Taipy/index.ts

```diff
@@ -74,6 +75,7 @@ export const getRegisteredComponents = () => {
             Table: Table,
             Toggle: Toggle,
             TreeView: TreeView,
+            ProgressBar: Progress,
```

Comment:
Can you change this for Progress: Progress ?
The key is used by JsxParser to find the component => it has to be the same has what is declared in factory.py